### PR TITLE
Automatic Build App

### PR DESCRIPTION
**Pull request by BLUE-DEVIL1134**
**I was facing problem in build,
so i made this change..**

### CODE REVIEW

File: docs/build.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    """SET PATH=%cd%\\ThirdParty\\Strawberry\\perl\\bin;%cd%\ThirdParty\\Python27;%cd%\\ThirdParty\\NASM;%cd%\\ThirdParty\\jom;%cd%\\ThirdParty\\cmake\\bin;%cd%\\ThirdParty\\yasm;%PATH%""")
+cmd("git clone --recursive https://github.com/telegramdesktop/tdesktop.git")
+cmd("cls")
+cmd("mkdir Libraries")
+cmd("cd Libraries")
+cmd("SET LibrariesPath=%cd%")
+cmd("git clone https://github.com/desktop-app/patches.git")
+cmd("cd patches")
+cmd("git checkout ddd4084")
+cmd("cd ..")
```
Comment (ratijas) [inline docs/build.py:29]: At this point, why would you use python at all? Besides, plain `*.bat` for cmd would benefit from not having to escape every special character in strings, like backslash (\\).

**Review by ratijas — COMMENTED** [review]
See my comment below.

File: docs/build.py (hunk 1 above)
Comment (BLUE-DEVIL1134) [reply docs/build.py:29]: well,
i am a python programmer and dont know batch much..
so i did this...

Comment (xvitaly) [reply docs/build.py:29]: All cmd() calls must be replaced by plain Python functions.

Comment (ratijas) [reply docs/build.py:29]: >  i am a python programmer and dont know batch much..
> so i did this...

Well, this is all literally a batch, but wrapped inside python `cmd` calls. It gains nothing from being wrapped like that.

**Comment by john-preston** [comment]
Sorry, I'd say that all you need to do is copy-paste all lines from original instruction into a .bat script - and that's all. No need for Python and new information, including your github and telegram profile links.

**Pull request closed without merging**